backend/dataset/tasks.py

```diff
@@ -11,7 +11,9 @@
 @shared_task(
     bind=True,
 )
-def upload_data_to_data_instance(self, dataset_string, pk, dataset_type, content_type):
+def upload_data_to_data_instance(
+    self, dataset_string, pk, dataset_type, content_type, deduplicate=False
+):

```

Comment:
Hi @ishvindersethi22 , shouldn't this be `True` by default?

Anyway ideally, when uploading the dataset file, we would also required a corresponding toggle button on the frontend for deduplicating the file that is being uploaded. If possible, kindly create a sub-ticket for that as well.

Reply:
Yes, we can pass it default as true from frontend?

Reply:
Sure, anything that works. But if the uploader intentionally wants to allow duplicates, it might be helpful to have the toggle to disable deduplication.

Reply:
Should I change it in the backend or will it be sent as True from the frontend?